packages/transaction-controller/src/utils/gas-fees.ts

```diff
@@ -169,13 +208,20 @@ function getGasPrice(request: GetGasFeeRequest): string | undefined {
   return undefined;
 }
 
-function getUserFeeLevel(request: GetGasFeeRequest): UserFeeLevel | undefined {
-  const { eip1559, initialParams, suggestedGasFees, txMeta } = request;
+function getUserFeeLevel(
+  request: GetGasFeeRequest,
+): UserFeeLevel | undefined | typeof CUSTOM_GAS_ESTIMATE {
+  const { eip1559, initialParams, savedGasFees, suggestedGasFees, txMeta } =
+    request;
 
   if (!eip1559) {
     return undefined;
   }
 
+  if (savedGasFees) {
+    return CUSTOM_GAS_ESTIMATE;
```

Comment:
If this is a valid option for the `userFeeLevel` property, should we just add it to the `UserFeeLevel` enum?

Reply:
Thanks for the heads up, I noticed that the value is already in the `UserFeeLevel.Custom`

Reply:
Fixed in https://github.com/MetaMask/core/pull/1966/commits/26ef4aa8c7777b023882bd83e9385dcadadce07d